homeassistant/components/backup/http.py

```diff
@@ -92,6 +95,8 @@ async def _send_backup_no_password(
     ) -> StreamResponse | FileResponse | Response:
         if agent_id in manager.local_backup_agents:
             local_agent = manager.local_backup_agents[agent_id]
+            # We don't need to check if the path exists, aiohttp.FileResponse will
+            # handle that
```

Comment:
What happens if it doesn't exist?

Reply:
I just moved this comment because it was in the wrong place after some previous changes.

I actually have on my TODO list to handle this ourselves, because the way `aiohttp.FileResponse` handles it is not great.(IIRC, it handles it in the sense it doesn't blow up, but there's no 404 because it only checks the file exists *after* we send the response).

That's for a different PR though.

Reply:
Ok